app/routes/manager/filter-tags/statuses.tsx

```diff
@@ -162,39 +134,13 @@ export const action: ActionFunction = async ({ request }) => {
   }
 };
 
-const GridEditToolbar = (props: gridEditToolbarProps) => {
-  const { setRows, createButtonText } = props;
-  const handleAddClick = () => {
-    const id = "new-value";
-    const newName = "";
-    const newStage = "";
-    setRows((prevRows) => {
-      if (prevRows.find((rowValue) => rowValue.id === "new-value")) {
-        return [...prevRows];
-      }
-      return [...prevRows, { id, name: newName, stage: newStage, isNew: true }];
-    });
-  };
-  return (
-    <GridToolbarContainer>
-      <Button
-        variant="contained"
-        color="primary"
-        startIcon={<AddIcon />}
-        onClick={() => handleAddClick()}
-        data-testid="testStatusCreate"
-      >
-        {createButtonText}
-      </Button>
-    </GridToolbarContainer>
-  );
-};
-
-export default function ProjectStatusDataGrid() {
+function ProjectStatusDataGrid() {
   const fetcher = useFetcher();
   const { statuses } = useLoaderData() as LoaderData;
   const createButtonText = "Create New Status";
   const [openDeleteModal, setOpenDeleteModal] = useState<boolean>(false);
+  const [openEditModal, setOpenEditModal] = useState<boolean>(false);
+  const [openCreateModal, setOpenCreateModal] = useState<boolean>(false);
   const [error, setError] = useState<string>("");
```

Comment:
we could remove this "error, setError" useState if we are not going to use it to trigger any action or display a message.